### PR TITLE
Generalized hand vise, renamed angular grinder 

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Bugfixes "Corrected regional names- vise and angle grinder"```

#### Purpose of change
Incorrect names were given for the hand vice and angular grinder (sic)
#### Describe the solution
Renamed angular grinder to angle grinder.

Changed Hand Vice to 'clamp', which more correctly described it. (We still don't have a real vise).
I left the ID's alone, no sense in changing those.
#### Describe alternatives you've considered
None.

#### Testing
String changes, none.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
These had been unconsciously bugging me for a while now, good spot. two thumbs up here.

**Comment by ymber** [comment]
Updating the IDs and migrating would be good for maintainability. IDs not matching names is irritating.

File: data/json/items/tools.json [hunk 1]
```diff
@@ -7381,10 +7381,10 @@
     "magazine_well": 2
   },
   {
-    "id": "hand_vice",
+    "id": "hand_vise",
```
Comment (ymber) [inline data/json/items/tools.json]: Why isn't this `clamp`?

File: data/json/items/tools.json [hunk 2]
```diff
@@ -7356,7 +7356,7 @@
   {
     "id": "angular_grinder",
```
Comment (ymber) [inline data/json/items/tools.json]: `angle_grinder`.

**Comment by tenmillimaster** [comment]
Alright, changes implemented.

**Comment by Rivet-the-Zombie** [comment]
Who in the heck calls it an 'angular grinder'?

**Pull request merged**